### PR TITLE
LUN-22: Setup fastlane

### PR DESCRIPTION
**Pull request by SteffiStuffel**

### CODE REVIEW

File: android/fastlane/Fastfile [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+default_platform(:android)
+
+platform :android do
+  desc "Runs all the tests"
```
Comment (klinzo) [inline android/fastlane/Fastfile:19]: Do we need this here?

File: android/fastlane/Fastfile [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  lane :test do
+    gradle(task: "test")
+  end
+
+  desc "Submit a new Beta Build to Crashlytics Beta"
+  lane :beta do
+    gradle(task: "clean assembleRelease")
+    crashlytics
```
Comment (klinzo) [inline android/fastlane/Fastfile:27]: I think we don't want crashlytics

File: docs/cicd.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+### Fastlane Setup
+- Make sure ruby is installed and `ruby --version` reports the correct version
+- run `bundle config set --local path 'vendor/bundle` to set the local path
+- run `bundle install` in the project root **AND** in `./android/` **AND** in `./ios/`.
+
+
+### Lanes
+To run a lane run `bundle exec fastlane [lane]`
+
+-
```
Comment (klinzo) [inline docs/cicd.md:25]:
```suggestion
```

File: android/fastlane/Appfile [hunk 4]
```diff
@@ -0,0 +1,2 @@
+json_key_file("") # Path to the json secret file - Follow https://docs.fastlane.tools/actions/supply/#setup to get one
```
Comment (klinzo) [inline android/fastlane/Appfile:1]: Strange, seems like we don't even have this in integreat.

Comment (klinzo) [inline android/fastlane/Appfile:1]: Strange, seems like we don't even have this in integreat.

**Comment by klinzo** [comment]
@maxammann maybe you could have a look at the basic setup and things we should tackle before others

File: android/fastlane/Appfile (hunk 4 above)
Comment (maxammann) [reply android/fastlane/Appfile:1]: This is fine if we only deploy one app here. This avoid setting it in the Fastfile

Comment (maxammann) [reply android/fastlane/Appfile:1]: This should be removed and directly passed via environment variables as base64 in the fastfile

**Review by maxammann — APPROVED** [review]
Looks fine for just getting fastlane into the projects. Steps which should follow:

* Build a release (including signing keys) for android locally via fastlane
* Same for iOS
* Run the same build in CI for Android and iOS
* Upload the APK and IPA to testing lanes
* (Setup Sentry if wanted)

File: android/fastlane/Appfile (hunk 4 above)
Comment (SteffiStuffel) [reply android/fastlane/Appfile:1]: done

Comment (SteffiStuffel) [reply android/fastlane/Appfile:1]: oke so i keep it? What about you comment above?

File: android/fastlane/Fastfile (hunk 2 above)
Comment (SteffiStuffel) [reply android/fastlane/Fastfile:27]: removed it.

**Review by klinzo — APPROVED** [review]

**Pull request merged**